app/build.gradle

```diff
@@ -58,4 +60,9 @@ dependencies {
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
     androidTestImplementation "androidx.compose.ui:ui-test-junit4:$compose_version"
     debugImplementation "androidx.compose.ui:ui-tooling:$compose_version"
+
+
+    //
+    implementation "androidx.navigation:navigation-compose:$nav_version"
+    implementation "androidx.compose.material:material-icons-extended:$compose_version"
```

Comment:
Just a note that `material-icons-extended` is a large library and can affect your APK size.